.github/workflows/docker.yaml

```diff
@@ -75,9 +78,10 @@ jobs:
           touch "/tmp/digests/${digest#sha256:}"
       -
         name: Upload digest
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         with:
-          name: digests
+          name: digests-${{ env.PLATFORM_NAME }}
+          overwrite: true
```

Comment:
Is this really necessary ?

Reply:
I think it's needed when you run the same pipeline twice